README.md

```diff
@@ -294,62 +333,51 @@ After installing:
     --outfile=bundle.js
     ```
 
-3. Make sure you include ES6 polyfills with your bundler if you need the broadest browser support. 
+3. Make sure you include ES6 polyfills with your bundler if you need the broadest browser support.
 
 #### Examples
 
+These simple examples should help you get started with using the Sign-in Widget. For complete end-to-end solutions, check out our [sample applications](#sample-applications).
+
 ##### SPA Application
 
-Although a `redirectUri` is required in the configuration, no redirection occurs using this flow. The Sign-in Widget will communicate with Okta and receive tokens directly.
+A Single Page Application (SPA) runs completely in the browser. SPA applications authenticate using client-side flows and store tokens and other data in browser-based storage.
```

Comment:
should we mention `other data`- it would be perceived as very broad.!!

Reply:
yes can probably leave "other data" out here